stanford_capx.install

```diff
@@ -439,6 +439,14 @@ function stanford_capx_update_7301() {
 
 }
 
+/**
+ * Upgrade path for PHP 7.2 for those who missed it in 7301.
+ */
+function stanford_capx_update_7302() {
+  // We could check the encryption method of the username and password variables here, but it's probably not worth the effort.
```

Comment:
CodeClimate doesn't like this because it's past the character limit. I saw this in PhpStorm but ignored it because it was only a few characters.

What say you - wrap to new line?